frontend/sw360-portlet/src/main/resources/META-INF/resources/html/search/view.jsp

```diff
@@ -102,6 +106,9 @@
 
 		</div>
 	</div>
+		<div class="row">
+			<h4><liferay-ui:message key="Note: If unchecked 'entire document' then search will be restricted to name, fullname, title." /></h4>
```

Comment:
please correct the indentation. user white-space instead of `tab`.

also add the key in language properties file.